homeassistant/components/comelit/__init__.py

```diff
@@ -12,7 +13,9 @@
 
 async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     """Set up Comelit platform."""
-    coordinator = ComelitSerialBridge(hass, entry.data[CONF_HOST], entry.data[CONF_PIN])
+    coordinator = ComelitSerialBridge(
+        hass, entry.data[CONF_HOST], entry.data[CONF_PORT], entry.data[CONF_PIN]
```

Comment:
Should this use `entry.data.get(CONF_PORT, DEFAULT_PORT)` ?

As existing entries won't have this key.

Reply:
Indeed, forgot about `__init__` at first, was concentrated on `config_flow`  :-(